platform/src/main/resources/patterns/log-patterns

```diff
@@ -7,7 +7,7 @@ SYSLOG %{SYSBASE} %{GREEDYDATA:Message}
 CLASS1 %{MAIN_CLASS}|%{GENERIC_CLASS}
 CLASS2 %{SPACE}%{MAIN_CLASS}|%{SPACE}%{GENERIC_CLASS}
 CLASS %{CLASS1}|%{CLASS2}
-HADOOP %{LONGTIMESTAMP:Timestamp} %{LOGLEVEL:Level} %{CLASS:Class} %{GREEDYDATA:Message}
+HADOOP %{LONGTIMESTAMP:Timestamp} %{LOGLEVEL:Level} %{GREEDYDATA:Message}
```

Comment:
If you change this pattern the tests should start failing.
Did you rerun them prior to submitting the PR?